server/config/environment/production.js

```diff
@@ -3,5 +3,10 @@ module.exports = {
   postgres: {
     // The DATABASE_URL variable should already include the username, password, host, port, and database name
     uri: process.env.DATABASE_URL
+  },
+  facebook: {
+    id: process.env.FACEBOOK_ID,
+    secret: process.env.FACEBOOK_SECRET,
+    callback: process.env.IP + '/Server/users/facebook/callback'
```

Comment:
@dmsakamoto / @stevenshuhyow should this callback property be defined as an environment variable just like id and secret?

Also, is this the correct path? Should it be /Server/auth/facebook/callback? Or should it be auth/facebook/callback? See lines 28 and 29 in routes.js for context.